src/requested-data/requested-data-reducer.ts

```diff
@@ -18,23 +18,36 @@ export function requestedData(
                 ...state,
                 electionType: action.electionType,
                 enableAutoSave: true,
+                dataLoaded: checkStateLoaded(state, action),
             };
         case RequestedDataActionType.INTIIALIZE_REQUESTED_VOTES:
             return {
                 ...state,
                 votes: action.votes,
+                dataLoaded: checkStateLoaded(state, action),
             };
         case RequestedDataActionType.INITIALIZE_REQUESTED_METRICS:
             return {
                 ...state,
                 metrics: action.metrics,
+                dataLoaded: checkStateLoaded(state, action),
             };
         case RequestedDataActionType.INIITALIZE_REQUESTED_PARAMETERS:
             return {
                 ...state,
                 parameters: action.parameters,
+                dataLoaded: checkStateLoaded(state, action),
```

Comment:
Why is this done in every action? I would have thought there would be a single action that updates after completion of the other actions.